packages/grid/src/useGrid.ts

```diff
@@ -0,0 +1,359 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import { useState, HTMLProps, HTMLAttributes, KeyboardEvent } from 'react';
+import { composeEventHandlers } from '@zendeskgarden/container-utilities';
+
+const gridKeys = ['ArrowLeft', 'ArrowRight', 'ArrowUp', 'ArrowDown', 'Home', 'End'];
+
+export interface IUseGridProps {
+  /** Enables wrapped keyboard navigation  */
+  wrap?: boolean;
```

Comment:
Do we know that row/column hopping is desirable on wrap? I can imagine cases where the expected wrapping behavior is to loop within the same row or column. @steue have we considered the implication from a design POV? Should the hook provide added flexibility? If so, what is the default behavior?

Reply:
It may be instructive that other Garden cursor containers automatically wrap (`Buttongroup`, `Selection`, `Tabs`) and don't offer this level of control to the consumer. We may want to follow suit here.

Reply:
The wrap behavior reflects WAI-ARIA's [Keyboard Interaction For Layout Grids](https://www.w3.org/TR/wai-aria-practices/#keyboard-interaction-for-layout-grids). The default behavior is non-wrapping, and the optional wrap "hops" to the next row/column. These don't have to be Garden's defaults, though. Happy to update the hook to match Garden design intention. @steue could you provide your thoughts on this one?

Reply:
Ah, staying strict to the specification is most important. Thanks for the clarification. Let's leave the behavior as-is.